ui/src/tag/TagSelector.tsx

```diff
@@ -114,11 +114,10 @@ export const TagSelector: React.FC<TagSelectorProps> = ({
             setSelectedEntries(selectedEntries.slice(0, selectedEntries.length - 1));
             setCurrentValue(event.ctrlKey ? '' : label(last));
         }
-        if (event.key === 'ArrowUp') {
+        if (event.key === 'ArrowUp' || (event.key === 'Tab' && event.shiftKey && open)) {
```

Comment:
Hey, thanks for the contribution but I don't think this is the expected behavior. The native html select behaves like the current behavior (tab goes to the next field instead of going to the next option in the select). So I rather not include this.

Reply:
Is this directed at the whole PR or just line 117? 

If you're against the whole setting, how about I add a setting for to toggle this, and by default it can be the current behaviour?

Either way no worries, I just keep pressing tab intending to choose an item from the list which is a slight annoyance.

Reply:
I rather not have this as a setting. This is about the tab behavior, the ESC change looks okay, tho we could probably remove the blur event, as it doesn't really to make sense to lose focus here.

Reply:
I have amended the commit to remove the blur and undid the tab change. I'll rename this PR as well.